app/app/src/main/java/at/tugraz/ist/sw20/mam3/cook/ui/recipes/RecipesFragment.kt

```diff
@@ -44,20 +46,41 @@ class RecipesFragment : Fragment() {
             fragmentTransaction.replace(R.id.nav_host_fragment, fragment)
             fragmentTransaction.addToBackStack(null)
             fragmentTransaction.commit()
+            val lv = lvRecipes;
+            lvRecipes.onItemLongClickListener = AdapterView.OnItemLongClickListener { parent, view, position, id ->
+                val item = parent.getItemAtPosition(position) as Recipe
+                registerForContextMenu(lv);
+                Toast.makeText(context!!, "Long click detected", Toast.LENGTH_SHORT).show()
+                true
+            }
         }
         return root
     }
-
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         recipesViewModel = ViewModelProvider(this).get(RecipesViewModel::class.java)
 
         val readyListener = object : DataReadyListener<List<Recipe>> {
             override fun onDataReady(data: List<Recipe>?) {
-                lvRecipes.adapter = RecipeAdapter(context!!, data ?: listOf())
+          //      lvRecipes.adapter = RecipeAdapter(context!!, data ?: listOf())
+                lvRecipes.adapter = RecipeAdapter(context!!, listOf(Recipe(1, "testrecipe", "description", "normal", 5, 10, false)))
```

Comment:
Please pull from develop and merge the current version of develop into your branch. You should be able to add new recipes and thus you don't need to create a static recipe here.